docker-compose.yml

```diff
@@ -16,11 +14,20 @@ services:
     env_file: .env.development
     volumes:
       - ./src:/app
-    ports:
```

Comment:
We don't need to export port now?

Reply:
The port is exposed from the `Dockerfile.django`, we don't need to add the port forwarding from the physical machine to the container as the django app will only be reachable via the nginx server
I've changed the nginx port to run on port 80 so after building the services they can directly go to http://localhost to access the app